docs/examples/quick-start-with-terraform.md

```diff
@@ -0,0 +1,302 @@
+
+You can inspect the deployed clusters by typing:
+
+```bash
+kind get clusters
+```
+
+You should see a couple of entries:
+
+```text
+milan
+rome
+```
+
+This means that two KinD clusters are deployed and running on your host.
+
+Then, you can simply inspect the status of the clusters.
+To do so, you can export the `KUBECONFIG` variable to specify the identity file for _kubectl_ and _liqoctl_, and then contact the cluster.
+
+By default, the kubeconfigs of the two clusters are stored in the current directory (`./rome-config`, `./milan-config`).
+You can export the appropriate environment variables leveraged for the rest of the tutorial (i.e., `KUBECONFIG` and `KUBECONFIG_MILAN`), and referring to their location, through the following:
+
+```bash
+export KUBECONFIG="$PWD/rome-config"
+export KUBECONFIG_MILAN="$PWD/milan-config"
+```
+
+```{admonition} Note
+We suggest exporting the kubeconfig of the first cluster as default (i.e., `KUBECONFIG`), since it will be the entry point of the virtual cluster and you will mainly interact with it.
+```
+
+### Install Liqo with Helm resource
+
+After creating the clusters KinD Terraform will install Liqo using the resource `helm_release` of the provider [Helm](https://registry.terraform.io/providers/hashicorp/helm/latest/docs) once correctly configured with the cluster config files.
+Once installation is completed the Liqo pods should be up and running on both clusters:
+
+```bash
+kubectl get pods -n liqo
+```
+
+```text
+NAME                                       READY   STATUS    RESTARTS   AGE
+liqo-auth-74c795d84c-x2p6h                 1/1     Running   0          2m8s
+liqo-controller-manager-6c688c777f-4lv9d   1/1     Running   0          2m8s
+liqo-crd-replicator-6c64df5457-bq4tv       1/1     Running   0          2m8s
+liqo-discovery-546fd594dc-kg6db            1/1     Running   0          2m8s
+liqo-gateway-78cf7bb86b-pkdpt              1/1     Running   0          2m8s
+liqo-metric-agent-5667b979c7-snmdg         1/1     Running   0          2m8s
+liqo-network-manager-5b5cdcfcf7-scvd9      1/1     Running   0          2m8s
+liqo-proxy-6674dd7bbd-kr2ls                1/1     Running   0          2m8s
+liqo-route-7wsrx                           1/1     Running   0          2m8s
+liqo-route-sz75m                           1/1     Running   0          2m8s
+liqo-webhook-7bb6dbf88d-2hvwx              1/1     Running   0          2m8s
+```
+
+### Liqo generate resource
+
+Once Liqo is installed in remote cluster, Terraform will generate the authentication parameters to peer the remote cluster with the local one.\
+The resource in charge of doing this is `liqo_generate` of [liqo](https://google.com) provider configured with config file or parameters of remote cluster:
```

Comment:
The link looks wrong.

Reply:
We forget to remove this `placeholder`. The link has to be updated when Liqo Provider will be published on Terraform registry.